### PR TITLE
i18n(zh-cn): Update translate about #6267

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Update the translation of the following files：
- [markdoc.mdx](https://github.com/withastro/docs/commit/b967bfc33c0336f68442ecda5c154e0e28089ef2)
- [mdx.mdx](https://github.com/withastro/docs/commit/09bb7d588f567d2132c31d0d4bdabc38e23be7c8)
- [vercel.mdx](https://github.com/withastro/docs/commit/5729c99838b8d948a6f1ee6a74c6cc7261a42c17)
- [markdown-content.mdx](https://github.com/withastro/docs/commit/26a0d7b4c7bbc99022a63d308453331452a19511)
- [styling.mdx](https://github.com/withastro/docs/commit/9a38dea302857b1012f60112dc116fda7c3a01f5)
- [typescript.mdx](https://github.com/withastro/docs/commit/70e806fe89e0f2e46342ba8979a0fe2dcf8460da)
- [auto.mdx](https://github.com/withastro/docs/commit/b2a57678cd61f8a71bf58dd143f4af5afd095c61)
- [manual.mdx](https://github.com/withastro/docs/commit/3291896d328cc1014c5a913b08d174ba60cc2345)
- [i18n.mdx](https://github.com/withastro/docs/commit/5caaf0fa68a1d9da1280153755c33415b8b79a6c)
- [api-reference.mdx](https://github.com/withastro/docs/commit/6eb52d417a450f3d0dee510a00b346e1cf6d332b)
- [cli-reference.mdx](https://github.com/withastro/docs/commit/ea7aa4d9afd251d630c0f6e1d4c9b755e292f5eb)
- [directives-reference.mdx](https://github.com/withastro/docs/commit/bb547d2410fda52423b47a13dd42a58d97e9e4a7)
- [Update add-view-transitons.mdx](https://github.com/withastro/docs/commit/240ff4ffba2a6efe1b70aa0f9198c256feb92fc4)

#### Related issues & labels (optional)

- #6267

### CODE REVIEW

**Comment by liruifengv** [comment]
i18n.mdx 还要一个 commit: https://github.com/withastro/docs/commits/main/src/content/docs/en/recipes/i18n.mdx?since=2023-12-29T14:50:04.000Z

**Comment by huyikai** [comment]
> i18n.mdx 还要一个 commit: https://github.com/withastro/docs/commits/main/src/content/docs/en/recipes/i18n.mdx?since=2023-12-29T14:50:04.000Z

已补充 commit

File: src/content/docs/zh-cn/recipes/i18n.mdx [hunk 1]
```diff
@@ -501,6 +502,7 @@ import StaticSsrTabs from '~/components/tabs/StaticSsrTabs.astro'
 - [Right-to-left Styling 101](https://rtlstyling.com/)
 
 ## 其他第三方库
-- [astro-i18next](https://github.com/yassinedoghri/astro-i18next) — An Astro integration for i18next including some utility components.
-- [astro-i18n](https://github.com/alexandre-fernandez/astro-i18n) — A TypeScript-first internationalization library for Astro.
-- [astro-i18n-aut](https://github.com/jlarmstrongiv/astro-i18n-aut) — An Astro integration for i18n that supports the `defaultLocale` without page generation. The integration is adapter agnostic and UI framework agnostic.
+- [astro-i18next](https://github.com/yassinedoghri/astro-i18next) — 为 i18next 提供的 Astro 集成，包括一些实用组件。
+- [astro-i18n](https://github.com/alexandre-fernandez/astro-i18n) — 为 Astro 设计的以 TypeScript 为主的国际化库。
+- [astro-i18n-aut](https://github.com/jlarmstrongiv/astro-i18n-aut) — i18n 的 Astro 集成，无需生成页面即可支持`defaultLocale`。该集成与适配器、UI 框架无关。
+- [paraglide](https://inlang.com/c/astro) - 一个专门为像 Astro 群岛这样的部分水合模式设计的完全类型安全的 i18n 库。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/recipes/i18n.mdx:508]:
```suggestion
- [paraglide](https://inlang.com/c/astro) — 一个专门为像 Astro 群岛这样的部分水合模式设计的完全类型安全的 i18n 库。
```
把这个符号统一吧。似乎是英文原版就有问题，先不管他。

**Review by liruifengv — APPROVED** [review]
LGTM

**Comment by liruifengv** [comment]
https://github.com/withastro/docs/pull/6208

api-reference 有新的修改。你也可以单独再提个PR。再关联这个PR

**Comment by liruifengv** [comment]
> #6208
> 
> api-reference 有新的修改。你也可以单独再提个PR。再关联这个PR

I have updated in https://github.com/withastro/docs/pull/6407. 

Merge the two PRs together to make sure they're detected correctly by lunaria.

**Comment by liruifengv** [comment]
> #6208
> 
> api-reference 有新的修改。你也可以单独再提个PR。再关联这个PR

I have updated in https://github.com/withastro/docs/pull/6407. 

Merge the two PRs together to make sure they're detected correctly by lunaria.

**Pull request merged**